rasa/core/actions/forms.py

```diff
@@ -308,7 +308,10 @@ def _user_rejected_manually(self, validation_events: List[Event]) -> bool:
     def _get_events_since_last_user_uttered(
         tracker: "DialogueStateTracker",
     ) -> List[SlotSet]:
+        # Why would latest message not be in events?
```

Comment:
`tracker.latest_message` is of type `Optional[UserUttered]` so in some tests there can be trackers without `UserUttered` events that could fail if this check is not in place.